app/javascript/article-form/elements/moreConfig.jsx

```diff
@@ -46,45 +41,57 @@ export default class MoreConfig extends Component {
       publishedField = (
         <div>
           <h4>Danger Zone</h4>
-          <button onClick={onSaveDraft}>Unpublish Post</button>
+          <button type="button" onClick={onSaveDraft}>
+            Unpublish Post
+          </button>
         </div>
       );
     }
     return (
       <div className="articleform__overlay">
         <h3>Additional Config/Settings</h3>
         <button
+          type="button"
           className="articleform__exitbutton"
           data-content="exit"
           onClick={onExit}
         >
           ×
         </button>
         <div>
-          <label>Canonical URL</label>
+          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+          <label htmlFor="canonicalUrl">Canonical URL</label>
           <input
             type="text"
+            id="canonicalUrl"
             value={passedData.canonicalUrl}
             name="canonicalUrl"
-            onKeyUp={this.props.onConfigChange}
+            onKeyUp={onConfigChange}
           />
         </div>
         <small>
-          Change meta tag <code>canonical_url</code> if this post was first
-          published elsewhere (like your own blog)
+          Change meta tag
+          <code>canonical_url</code>
+          if this post was first published elsewhere (like your own blog)
         </small>
         <div>
-          <label>Series Name</label>
+          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
```

Comment:
it says that the label should have an associated input element. I've done everything according to the [documentation](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md) but I'm still getting the error... Any idea?

Reply:
Hey, in my PR #4253, I have config eslint to accept that case.

The rule forces us to have **both** the nesting and id (documentation unclear), due to browser compatibility. But now we should only need 1 of them.

I'm wondering if I should have the commit as a seperate PR.

Reply:
It's totally fine with me. Can we have a core team member to agree on this?

Reply:
I would fix the eslint config and remove the two "disable next line" rules

Reply:
Alright, so I'm gonna include this fix in my PR. @hoangvvo or I will resolve the merge conflict according to whose branch gets merged first.